### DISCUSSION TITLE
How do I create initial user for supavisor?

### DISCUSSION THREAD

**Discussion by hgezim** [supabase/supabase#22175] [Questions]
[This issue](https://github.com/supabase/supavisor/issues/202#issue-1978724103) covers creating the tenant in supavisor.

It uses an Authorization token and I'm not sure how to get such a token when I don't have a user for supavisor.

Ideas?

**Comment by silentworks** [comment]
Looking at the issue it would seem they are just using the `anon_key` as the token.
